rocAL/source/parameters/parameter_random_crop_decoder.cpp

```diff
@@ -27,14 +27,19 @@ THE SOFTWARE.
 // Initializing the random generator so all objects of the class can share it.
 thread_local std::mt19937 RocalRandomCropDecParam::_rand_gen(time(0));
 
-RocalRandomCropDecParam::RocalRandomCropDecParam(
```

Comment:
This initialize function is not used anywhere else? Why is the init removed?

Reply:
It was moved to the .h file and the constructor was changed for inheriting from CropParam class